### DISCUSSION TITLE
Mastodon main web 1 and sidekiq  docker container keep restarting

### DISCUSSION THREAD

**Discussion by anasmilhem** [mastodon/mastodon#18659] [Q&A]
Hey!
I am trying to run a local mastodon docker container on my macbook. But it seems like the two container keep restarting and I'm not able to understand why? Any help on this would be great. 

Thank You!

**Comment by Cyb3r14n-Security** [comment]
did you find an answer?

**Comment by MrGeneration** [comment]
So what do your logs say when you run `docker compose logs -f` ?
Sounds like dependencies of the services are not fulfilled to the moment the container is starting
